metadata-ingestion/src/datahub/ingestion/source/looker/lookml_concept_context.py

```diff
@@ -251,19 +256,20 @@ def resolve_extends_view_name(
     def get_including_extends(
         self,
         field: str,
+        extends_only: bool = False,
     ) -> Optional[Any]:
         extends = list(
             itertools.chain.from_iterable(
                 self.raw_view.get("extends", self.raw_view.get("extends__all", []))
             )
         )
 
-        # First, check the current view.
-        if field in self.raw_view:
+        # If extends_only is false, first check the current view, and then fall back to the parent view.
+        if extends_only is False and field in self.raw_view:
```

Comment:
I'm a bit confused by this - given that the method is called `get_including_extends`, I would expect it to handle all the merge logic internally.

you can add a list of field names that require merge logic, and the rest can continue to use override logic as the current method does

Reply:
Done